profiling/Cargo.toml

```diff
@@ -22,6 +22,7 @@ crossbeam-channel = { version = "0.5", default-features = false, features = ["st
 datadog-alloc = { git = "https://github.com/DataDog/libdatadog", tag = "v15.1.0" }
 datadog-profiling = { git = "https://github.com/DataDog/libdatadog", tag = "v15.1.0" }
 ddcommon = { git = "https://github.com/DataDog/libdatadog", tag = "v15.1.0" }
+datadog-library-config-ffi = { git = "https://github.com/DataDog/libdatadog", tag = "v16.0.3" } # FIXME: version
```

Comment:
The `#FIXME` is related to the other `libdatadog` dependencies being `v15.1.0` while this is `v16.0.3` I guess? Is this something that needs fixing for this PR or is it unrelated and just "should be synced with the others"?

Reply:
> The #FIXME is related to the other libdatadog dependencies being v15.1.0 while this is v16.0.3 I guess?

Yes, you're right

> Is this something that needs fixing for this PR or is it unrelated and just "should be synced with the others"?

I don't think it needs fixing, but I wanted to get your opinion on this.
Because from now on, it's a new constraint: all 3 extensions need to use a compatible version of `library-config`.

Reply:
I updated the comment to `# Must be kept in sync with other extensions using the zend_abstract_interface/config library`.